Formula/h/hdf5.rb

```diff
@@ -1,10 +1,10 @@
 class Hdf5 < Formula
   desc "File format designed to store large amounts of data"
   homepage "https://www.hdfgroup.org/HDF5"
```

Comment:
May want to update homepage as it is currently a PDF.

Reply:
Done